prow/plugins/projectmanager/projectmanager_test.go

```diff
@@ -0,0 +1,678 @@
+					ID:    2,
+					State: "open",
+				},
+				Repo: github.Repo{
+					Name: "someRepo",
+					Owner: github.User{
+						Login: "otherOrg",
+					},
+				},
+			},
+			expectedColumnCards: map[int][]github.ProjectCard{
+				1: {
+					{
+						ContentID:   2,
+						ContentType: "PullRequest",
+					},
+				},
+			},
+		},
+		{
+			name: "add pull request using column name into org and repo projects",
+			gc: &fakegithub.FakeClient{
+				IssueLabelsAdded:   labels,
+				IssueLabelsRemoved: []string{},
+				RepoProjects: map[string][]github.Project{
+					"testOrg/*": {
+						{
+							Name: "testProject",
+							ID:   1,
+						},
+					},
+					"testOrg/testRepo2": {
+						{
+							Name: "testProject2",
+							ID:   2,
+						},
+					},
+				},
+				OrgProjects: map[string][]github.Project{},
+				ProjectColumnsMap: map[string][]github.ProjectColumn{
+					"testProject": {
+						{
+							Name: "testColumn2",
+							ID:   2,
+						},
+						{
+							Name: "testColumn",
+							ID:   1,
+						},
+					},
+					"testProject2": {
+						{
+							Name: "testColumn2",
+							ID:   4,
+						},
+						{
+							Name: "testColumn",
+							ID:   3,
+						},
+					},
+				},
+				ColumnCardsMap: map[int][]github.ProjectCard{},
+				ColumnIDMap:    columnIDMap,
+			},
+			projectManager: plugins.ProjectManager{
+				OrgRepos: map[string]plugins.ManagedOrgRepo{
+					"testOrg": {
+						Projects: map[string]plugins.ManagedProject{
+							"testProject": {
+								Columns: []plugins.ManagedColumn{
+									{
+										Name:   "testColumn",
+										State:  "open",
+										Org:    "otherOrg",
+										Labels: []string{"label1"},
+									},
+								},
+							},
+						},
+					},
+					"testOrg/testRepo2": {
+						Projects: map[string]plugins.ManagedProject{
+							"testProject2": {
+								Columns: []plugins.ManagedColumn{
+									{
+										Name:   "testColumn2",
+										State:  "open",
+										Org:    "otherOrg",
+										Labels: []string{"label1", "label2"},
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			pe: github.PullRequestEvent{
+				Action: github.PullRequestActionOpened,
+				Number: 1,
+				PullRequest: github.PullRequest{
+					ID:    2,
+					State: "open",
+				},
+				Repo: github.Repo{
+					Name: "someRepo",
+					Owner: github.User{
+						Login: "otherOrg",
+					},
+				},
+			},
+			expectedColumnCards: map[int][]github.ProjectCard{
+				4: {
+					{
+						ContentID:   2,
+						ContentType: "PullRequest",
+					},
+				},
+				1: {
+					{
+						ContentID:   2,
+						ContentType: "PullRequest",
+					},
+				},
+			},
+		},
+	}
+	for _, c := range cases {
+		t.Run(c.name, func(t *testing.T) {
+			err := handlePR(c.gc, c.projectManager, logrus.NewEntry(logrus.New()), c.pe)
+			if err != nil {
+				if c.expectedError != nil && c.expectedError.Error() == err.Error() {
+					return
+				}
+				t.Fatalf("handlePR error: %v", err)
+			}
```

Comment:
Please fail the test if we expect an error and don't see one.

Reply:
fixed.